revscoring/features/wikitext/datasources/parsed.py

```diff
@@ -0,0 +1,239 @@
+import re
+
+import mwparserfromhell
+
+from ....datasources import Datasource
+from ....datasources.meta import filters, mappers
+
+
+class Revision:
```

Comment:
As discussed in chat, this is only necessary python 2.x.  In python 3, inheritance from `object` is implied.